### ISSUE TITLE
[Drag & Drop] Dragged elements do not handle scrollable containers correctly

### ISSUE THREAD

**Issue by vin0401** [pimcore/studio-ui-bundle#411]
When the tree container is scrollable, the container is scrolled when the cursor is at the bottom or top of the browser window.
This is fine when the cursor is inside of the current container, but unfortunately the same happens when the cursor is positioned over another container outside the tree.

**Comment by markus-moser** [comment]
@vin0401 Still relevant?

**Comment by vin0401** [comment]
yep, still relevant. But right now, it seems like that there is not any scroll at all happening.

**Comment by markus-moser** [comment]
I think we solved this by refactoring it to the native browser event solution -> lets close it.
